engine/robot_state.py

```diff
@@ -102,6 +103,11 @@ def __init__(self, **kwargs):
         self.width = kwargs.get("width", 700)
         self.length = kwargs.get("length", 700)
         self.time_step = kwargs.get("time_step", 1)
+        # Based on our current code, 4 = Roomba mode, anything else is regular traversal 
+        # Probably should update this in the future
+        self.control_mode = kwargs.get("control_mode", 2) 
```

Comment:
why do we need this? It looks like its only set in execute_traversal and not called anywhere else

Reply:
i think we wanted to give users the flexibility to choose a control mode i.e standard lawnmower or roomba traversal, but it could be done in a better way - this might be something for a different PR tho

Reply:
we have a whole control mode file LOL

Reply:
will update to reflect this!